lib/fenrir_view/components/fenrir_view_accessibility_issue/_fenrir_view_accessibility_issue.html.erb

```diff
@@ -0,0 +1,16 @@
+<article class="fenrir-view-accessibility-issue">
+  <h5 data-audit-id="<%= @component.id %>"
+      data-display-mode="<%= @component.score_display_mode %>"
+      data-score="<%= @component.score %>">
+    <%= @component.title %>
+    <%= " (#{@component.score_as_percentage}% pass, #{@component.failure_as_percentage}% failure)" if @component.show_score? %>
```

Comment:
This feels like it could be a method. But I don't think it's a blocker if you'd rather keep the component class a bit cleaner

Reply:
Something like

```ruby
def title
[
  issue.title,
  ("(#{@component.score_as_percentage}% pass, #{@component.failure_as_percentage}% failure)" if @component.show_score?)
].compact.join(' ')
end
```

Reply:
Yup, agree, much nicer.